compiler/plugins/input/StableHLO/stablehlo-iree/Conversion/Preprocessing/test/canonicalization.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt --iree-stablehlo-canonicalize --allow-unregistered-dialect --split-input-file %s | FileCheck %s
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-stablehlo-canonicalize))" --allow-unregistered-dialect --split-input-file %s | FileCheck %s
```

Comment:
I much prefer when tests can omit `--pass-pipeline`, since I can never remember off the top of my head what syntax to use like here. Oh well :/

Reply:
the inconsistency has always bothered me - now at least we're consistently on the most featureful form